packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_ExecutionManagerWrapper.sol

```diff
@@ -186,4 +188,30 @@ library Lib_ExecutionManagerWrapper {
             }
         }
     }
+
+    /**
+     * Performs an ovm interaction and the necessary safety checks while passing
+     * through a specified gas limit for the delegatecall.
+     * @param _calldata Data to send to the OVM_ExecutionManager (encoded with sighash).
+     * @return Data sent back by the OVM_ExecutionManager.
+     */
```

Comment:
Is it worth refactoring a bit to combine this function with `_callWrapperContract`? They are nearly the same, the above would just call this one with `gasleft` as the second param.

Reply:
Yes, worth combining IMO. Can just use the same name with an overload.